server/service/src/sync/translations/program_requisition_settings.rs

```diff
@@ -27,14 +27,7 @@ pub struct LegacyListMasterRow {
 #[derive(Deserialize, Clone)]
 struct LegacyProgramSettings {
     #[serde(rename = "storeTags")]
-    store_tags: LegacyStoreTagAndProgramName,
-}
-
-#[derive(Deserialize, Clone)]
-struct LegacyStoreTagAndProgramName {
-    // HashMap key is the program name
-    #[serde(flatten)]
-    tags: HashMap<String, LegacyProgramSettingsStoreTag>,
```

Comment:
@roxy-dao was there any reason to use flatten? and this struct?

Reply:
Not one I can remember anymore XD

Reply:
I find the new version simpler, so if you don't object I would use this now...